pipeline/terraform/modules/stack/service_id_minter.tf

```
@@ -45,8 +52,46 @@ module "id_minter" {
   cpu    = 2048
   memory = 4096
 
-  min_capacity = var.min_capacity
-  max_capacity = local.max_capacity
+  // TODO: Temporary disable the ECS id_minter, while we ensure the lambda is working
+  // Delete this block once we are confident the lambda is working.
+  min_capacity = 0 //var.min_capacity
+  max_capacity = 0 //local.max_capacity
 
   fargate_service_boilerplate = local.fargate_service_boilerplate
 }
+
+module "id_minter_lambda" {
+  source = "../pipeline_lambda"
+
+  pipeline_date = var.pipeline_date
+  service_name  = "${local.namespace}_batcher"
```

Should this be id_minter?
```suggestion
  service_name  = "${local.namespace}_id_minter"
```